maxmin/max3/max3.go

```diff
@@ -0,0 +1,20 @@
+package main
+
+/* (maxmin/max2) a program that calculates the maximum of three numbers.*/
+
+import (
+	"fmt"
+	"math"
+)
+
+func main() {
+	fmt.Println("The program finds the maximum of three numbers.")
+	fmt.Println("Enter three numbers:")
+	var a, b, c float64
+	fmt.Scan(&a, &b, &c)
+	//max := math.Max(math.Max(a, b), c)
+	max1 := (a + b + math.Abs(a-b)) / 2
+	max2 := (b + c + math.Abs(b-c)) / 2
+	max := (max1 + max2 + math.Abs(max1-max2)) / 2
```

Comment:
this should be done with two actions, not with three. Your current code does this:
1) find max between first two numbers
2) find max between last two numbers
3) find max between previous two max

this is correct, but it does one extra comparison. please try to optimize this. consider using the fact that `max(a, b, c) = max(max(a, b), c)`